apps/docs/src/app/page.tsx

```diff
@@ -122,14 +122,14 @@ export default function HomePage() {
           or learn more about how the Design System Building Block can benefit
           your department.
         </Paragraph>
-        <Link
+        {/* <Link
```

Comment:
unecessary comment

Reply:
I left there on purpose because the form link stopped working recently.
We might have to put that back